tests/manage/z_cluster/cluster_expansion/test_add_capacity_ms.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import pytest
+
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import cal_md5sum
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import config
+from ocs_ci.utility.rosa import configure_managed_service_size, get_managed_service_size
+from ocs_ci.helpers.managed_services import verify_provider_topology
+from ocs_ci.framework.testlib import (
+    ignore_leftovers,
+    ManageTest,
+    ms_provider_and_consumer_required,
+    tier1,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@ignore_leftovers
+@ms_provider_and_consumer_required
+class TestAddCapacityMS(ManageTest):
+    """
+    Automates adding variable capacity to the cluster
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        """
+        Prepare pods for the test and add finalizer.
+
+        """
+        self.provider_cluster_index = config.get_provider_index()
+        self.consumer_indexes = config.get_consumer_indexes_list()
+
+    def test_add_capacity_managed_service(
+        self, pvc_factory, pod_factory, teardown_project_factory
+    ):
+        """
+        Test to add capacity on MS
+
+        1.Crate new project on Consumer cluster
+        2.Create PVC and FIO POD
+        3.Generate 1G data
+        4.check md5sum on FIO POD
+        5.Get current size via rosa cmd on Provider Cluster
+        6.Configure New size via rosa cmd on Provider Cluster
+        7.Verify all osd pods are running [based on size] on Provider Cluster
+        8.Check Ceph Status
+        9.Create new PVC on Consumer Cluster and verify it moved to bound state
+        10.Verify md5sum is equal to step 4
+
+        """
+        SIZE = {
+            "4": "8",
+            "8": "12",
+            "12": "16",
+            "16": "20",
+            "20": "48",
+            "48": "69",
+            "96": "96",
+        }
+        config.switch_ctx(self.consumer_indexes[0])
+        self.project_obj = helpers.create_project()
+        teardown_project_factory(self.project_obj)
+
+        logger.info("Create PVC1 CEPH-RBD, Run FIO and get checksum")
+        pvc_obj_rbd1 = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            project=self.project_obj,
+            size=2,
+            status=constants.STATUS_BOUND,
+        )
+        pod_rbd1_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
```

Comment:
Why is interface filesystem if rbd pvc is used (pvc_obj_rbd1)?

Reply:
the pod uses fs to run fio. however, the PVC is rbd.